app/controllers/api/v1/graphql_controller.rb

```diff
@@ -66,7 +66,7 @@ def format_error_message(e)
       private
 
       def authenticate_graphql_user
-        params[:query].to_s.match(/^((query )|(mutation[^\{]*{\s*(resetPassword|changePassword|resendConfirmation)))/).nil? ? authenticate_user! : authenticate_user
+        params[:query].to_s.match(/^((query )|(mutation[^\{]*{\s*(resetPassword|changePassword|userDisconnectLoginAccount|)))/).nil? ? authenticate_user! : authenticate_user
```

Comment:
Why you removed the `resendConfirmation` mutation from here?

Reply:
Removed by mistake will add it